### PR TITLE
Feature/cleaning up part1

### PR DESCRIPTION
**Pull request by PAKovalev**

### CODE REVIEW

File: packages/app/src/shared/hooks/useBlockchain.tsx [hunk 1]
```diff
@@ -180,85 +179,3 @@ export const useBlockchain = ({requiredAccountAddress}: UseBlockchainPropsInterf
     getTokens
   };
 };
-
```
Comment (dmitry-yudakov) [inline packages/app/src/shared/hooks/useBlockchain.tsx:183]: it's more convenient to keep them in one file as they're very tightly coupled... but ok :)

File: packages/ui-kit-storybook/src/atoms/WalletHash/WalletHash.tsx [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
+  icon?: IconNameType;
   hash: string;
 }
 
 const WalletHash: FC<WalletHashPropsInterface> = ({icon, hash}) => {
   return (
-    <styled.WalletContainer>
-      <IconSvg name="talisman" size="l" />
+    <styled.WalletContainer data-testid="WalletHash-test" className={cn(!icon && 'noIcon')}>
+      {icon && <IconSvg name="talisman" size="l" />}
```
Comment (dmitry-yudakov) [inline packages/ui-kit-storybook/src/atoms/WalletHash/WalletHash.tsx:17]: `name={icon}` ?

**Review by dmitry-yudakov — APPROVED** [review]
Approved in general with 1 comment that should probably be addressed

File: packages/ui-kit-storybook/src/atoms/WalletHash/WalletHash.tsx (hunk 2 above)
Comment (PAKovalev) [reply packages/ui-kit-storybook/src/atoms/WalletHash/WalletHash.tsx:17]: There is a separate ticket for it )

**Pull request merged**